python/unicornafl/__init__.py

```diff
@@ -191,9 +125,27 @@ def uc_afl_fuzz(uc: Uc,
     cb2 = ctypes.cast(UC_AFL_VALIDATE_CRASH_CB(
         _validate_crash_cb), UC_AFL_VALIDATE_CRASH_CB)
 
-    err = _uc2afl.uc_afl_fuzz(uc._uch, input_file.encode("utf-8"), cb1, ctypes.cast(
-        exits_array, ctypes.c_void_p), exits_len, cb2, always_validate, persistent_iters, ctypes.cast(idx, ctypes.c_void_p))
-
+    if isinstance(input_file, str):
+        input_file = input_file.encode('utf-8')
```

Comment:
```suggestion
        input_file = input_file.encode('utf-8') + b"\x00"
```

This does not seem sufficient - `python_str.encode` does *not* garantee null termination of the string. I believe that that we must manually add a single zero byte to ensure that the string can be decoded by CStr::from_ptr(). Another valid path would be `ctypes.create_string_buffer` but that would not improve readability.